browser/app/js/App.js

```diff
@@ -34,13 +34,15 @@ const AuthorizedRoute = ({ component: Component, ...rest }) => (
   />
 )
 
-export const App = ({ match }) => (
-  <Switch>
-    <AuthorizedRoute exact path={match.url} component={Browser} />
-    <Route path={`${match.url}/login`} component={Login} />
-    <AuthorizedRoute path={`${match.url}/:bucket/*`} component={Browser} />
-    <AuthorizedRoute path={`${match.url}/:bucket`} component={Browser} />
-  </Switch>
-)
+export const App = ({ match }) => {
```

Comment:
Is there a particular reason why we have `match` here? As far as I can see we don't use it anywhere apart from the unit test

Reply:
Not being used, will remove it